js/script.js

```diff
@@ -424,11 +472,23 @@ function debugMsg(...args) {
     }
 }
 
-function errorMsg(text) {
+function errorMsg(text, forwardLink=null) {
+    // regular log with red Error: prefix
     logMsg('<span class="error-message">Error:</span> ' + text);
-    console.error(text);
```

Comment:
Yep I agree, it's super important and helpful! It's still happening on the next few lines:
```javascript
    // strip html for console and alerts
    const strippedText = text.replaceAll(/<.*?>/g, "")
    // all errors go to the browser dev console
    console.error(strippedText);
```

I used HTML in some of my errors to make them read better since we were improving the UX for lay persons, but of course that only looks good for the in-document stuff, just noise for console and alerts.